backend/src/main/scala/cromwell/backend/ExecutionHash.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.backend
+
+import java.util.UUID
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ExecutionHash {
+  // TODO: PBE: ideally hashes should be deterministic
```

Comment:
I understand the theory, but I have been leaving non-PBE `TODO` comments around "integration" points for code that's probably fine... but if not provides a hint to future-me as to a fix.